src/DependencyInjection/AlgoliaSearchExtension.php

```diff
@@ -81,7 +87,8 @@ public function load(array $configs, ContainerBuilder $container)
             ]
         ))->setPublic(true);
 
-        $container->setDefinition('search.service', $searchServiceDefinition);
+        $container->setDefinition('search.service', $searchServiceDefinition->setPublic(true));
+        $container->setDefinition('search.service_for_atomic_reindex', $searchServiceDefinitionForAtomicReindex);
```

Comment:
Just to confirm, this service is internal? People can't actually use it right?

Reply:
It's private, not internal